combat-trainer.lic

```diff
@@ -952,6 +1001,8 @@ class SpellProcess
     @tk_spell = @offensive_spells.find { |spell| spell['abbrev'] =~ /tkt|tks/i }
     drop_tkt_ammo
     Flags.add('need-tkt-ammo', 'There is nothing here that can be thrown')
+    
+    @regalia_spell = get_data('spells').spell_data['Regalia'] if (@regalia_array && @regalia_spell.nil?)
```

Comment:
❓ Why not do this when `@regalia_spell` and `@regalia_array` are initialized (around line 975). Then this only has to run once.

Reply:
I put it down there since defining a regalia_spell was performing a similar role as the part of initialize defining a TK spell, which happens right next to it.  I can move it, but although it might not look like it on review it IS still inside SpellProcess' initialize so it only happens once anyway.